can/interfaces/socketcan/socketcan.py

```diff
@@ -536,6 +536,7 @@ def capture_message(
     is_remote_transmission_request = bool(can_id & CAN_RTR_FLAG)
     is_error_frame = bool(can_id & CAN_ERR_FLAG)
     is_fd = len(cf) == CANFD_MTU
+    is_tx = bool(msg_flags & socket.MSG_DONTROUTE)
```

Comment:
I would add a comment explaining the use of this flag:
`# MSG_DONTROUTE: set when the received frame was created on the local host.`

Reply:
Done